src/Raven.Server/Smuggler/SmugglerHandler.cs

```diff
@@ -28,10 +32,103 @@ public Task PostExport()
+                        }
+
+                        switch (state.CurrentTokenType)
+                        {
+                            case JsonParserToken.Null:
+                                break;
+                            case JsonParserToken.False:
+                                break;
+                            case JsonParserToken.True:
+                                break;
+                            case JsonParserToken.String:
+                                if (objectDepth == 1)
+                                    unsafe
+                                    {
+                                        operateOnType = new LazyStringValue(null, state.StringBuffer, state.StringSize, context).ToString();
+                                    }
+                                break;
+                            case JsonParserToken.Float:
+                                break;
+                            case JsonParserToken.Integer:
+                                break;
+                            case JsonParserToken.Separator:
+                                break;
+                            case JsonParserToken.StartObject:
+                                if (objectDepth == 1)
+                                {
+                                    using (var builder = new BlittableJsonDocumentBuilder(context, BlittableJsonDocumentBuilder.UsageMode.ToDisk, "f", parser, state))
+                                    {
+                                        builder.ReadNestedObject();
+                                        while (builder.Read() == false)
+                                        {
+                                            var read = await stream.ReadAsync(buffer, 0, buffer.Length);
+                                            if (read == 0)
+                                                throw new EndOfStreamException("Stream ended without reaching end of json content");
+                                            parser.SetBuffer(buffer, read);
+                                        }
+                                        builder.FinalizeDocument();
+                                        using (var reader = builder.CreateReader())
+                                        using (var tx = context.OpenWriteTransaction())
+                                        {
+                                            BlittableJsonReaderObject metadata;
+                                            if (reader.TryGet(Constants.Metadata, out metadata) == false)
+                                                throw new InvalidOperationException("A document must have a metadata");
+                                            // We are using the @id here and not @key in order to be backward compatiable with old export files.
+                                            string key;
+                                            if (metadata.TryGet(Constants.MetadataDocId, out key) == false)
```

Comment:
Constants.MetadataDocId is "@id", I'll change the variable name to also be id and not key